roles/foreman_repositories/tasks/redhat.yml

```diff
@@ -18,9 +18,28 @@
   tags:
     - packages
 
-- name: 'Setup Katello {{ foreman_repositories_katello_version }} Repository'
-  yum:
-    name: https://fedorapeople.org/groups/katello/releases/yum/{{ foreman_repositories_katello_version }}/katello/el{{ ansible_distribution_major_version }}/x86_64/katello-repos-latest.rpm
-    disable_gpg_check: true
-    state: present
-  when: foreman_repositories_katello_version is defined
+- when: foreman_repositories_katello_version is defined
```

Comment:
@evgeni I know this is rather ugly. The first run of it worked. Trying again as well. The problem I face is that I am blocked making progress on this PR and others without this approach given CI won't test what we need it to test. Thoughts?